airflow/dags/ingestion/ask-astro-load-cosmos-docs.py

```diff
@@ -0,0 +1,58 @@
+import os
+from datetime import datetime
+
+from include.tasks.extract.cosmos_docs import extract_cosmos_docs
+from include.utils.slack import send_failure_notification
+
+from airflow.decorators import dag, task
+from airflow.providers.weaviate.operators.weaviate import WeaviateDocumentIngestOperator
+
+ask_astro_env = os.environ.get("ASK_ASTRO_ENV", "dev")
+
+_WEAVIATE_CONN_ID = f"weaviate_{ask_astro_env}"
+WEAVIATE_CLASS = os.environ.get("WEAVIATE_CLASS", "DocsDev")
```

Comment:
Should this env var be documented in the README or is this really an internal env var (or maybe shouldn't be configurable)?

Reply:
I think these env vars have existed back way ask astro was first created. I just happen to have been using it as well in this new file. The repo has references to them at a lot of places. I believe it is not an internal env var though, as technically anyone cloning it can set this up.